lib/core/network/api_service.dart

```diff
@@ -505,6 +506,27 @@ class APIService {
     return response.statusCode == 200;
   }
 
+  Future<bool> editChildAllowance(String childGUID, int allowance) async {
+    final url =
+        Uri.https(apiURL, '/givtservice/v1/ChildProfile/$childGUID/allowance');
```

Comment:
Minor backend change. Can you change 'childprofile' to 'profile' in the url? I just made some changes with Pedro to streamline things
Thanks 😇

 /givtservice/v1/profile/$childGUID/allowance

Reply:
Done